tools/configure.in

```diff
@@ -166,7 +166,7 @@ for file in $boost_base/libs/config/test/boost_no*.ipp; do
    basename=`echo $file | $SED 's/.*boost_\(.*\)\.ipp/\1/'`
    macroname=`cat $file | grep '^//[[ 	]]*MACRO:' | $SED 's/.*MACRO:[[ 	]]*\([[_A-Z0-9]]*\).*/\1/'`
    title=`cat $file | grep '^//[[ 	]]*TITLE:' | $SED 's/.*TITLE:[[ 	]]*\([[^ 	]].*\)/\1/'`
-   namespace=`echo $macroname | tr [[A-Z]] [[a-z]]`
+   namespace=`echo $macroname | tr $as_cr_LETTERS $as_cr_letters`
```

Comment:
To be honest, I don't know. It somehow makes its way into the "configure" script - I guess it might be code inserted by the Autotools? Should I change this to `ABCDEFGHIJKLMNOPQRSTUVWXYZ` instead to avoid a dependency on wherever this comes from?